craft_providers/lxd/launcher.py

```diff
@@ -95,6 +101,7 @@ def _create_instance(  # noqa: PLR0913
     project: str,
     remote: str,
     lxc: LXC,
+    prepare_instance: "Callable[[Executor], None] | None" = None,
```

Comment:
Can you add a `:param:` in the docstring?

edit: This is in a few more functions throughout the library - do you mind updating the docstrings for all of them? I won't leave half a dozen of the same comments.

Reply:
Added missing params.